pkg/config/load.go

```diff
@@ -90,12 +101,23 @@ func InfoFromPath(configFilePath string) (*Info, error) {
 		branch = branch[:i]
 	}
 
+	pConfig := Prowgen{}
+	b, err := ioutil.ReadFile(filepath.Join(configSpecDir, prowgenConfigFile))
+	if err != nil && !os.IsNotExist(err) {
+		return nil, fmt.Errorf("prowgen config found but couldn't read the file: %v", err)
+	} else {
```

Comment:
This should be `else if err != nil {` otherwise it is the same like before, would `Unmarshal(b, ...)` even if it has whatever value when `ReadFile` failed with NotExist.

Reply:
thanks nice catch :)